### PR TITLE
Record public ip for objc

### PR DESCRIPTION
**Pull request by mitsuhiko**

### CODE REVIEW

**Comment by mitsuhiko** [comment]
@dcramer that's one way to make it work, the other would be to set the ip to a sentinel value (maybe `<auto>`) and let the SDKs send that value and we switch it for the remote addr on the client. Might be less magical.

**Comment by dcramer** [comment]
I like the idea of an auto ip. How about {{default}} like fingerprint? Or at least {{auto}} so it uses the same kind of template syntax.

**Comment by mitsuhiko** [comment]
Yeah. Can do that.

**Comment by mitsuhiko** [comment]
Memo to myself: if we go with this I need to update the docs.

**Comment by mitsuhiko** [comment]
@getsentry/infrastructure can someone review the changes here?

File: src/sentry/coreapi.py [hunk 1]
```diff
@@ -39,6 +39,10 @@
 from sentry.utils.validators import is_float
 
 LOG_LEVEL_REVERSE_MAP = dict((v, k) for k, v in LOG_LEVELS.iteritems())
+IP_INTERFACE_PATHS = [
```
Comment (dcramer) [inline src/sentry/coreapi.py]: unused

Comment (mitsuhiko) [inline src/sentry/coreapi.py]: thanks. killing.

**Comment by mitsuhiko** [comment]
@dcramer killed it. good for merge?

**Comment by dcramer** [comment]
:+1:

**Pull request merged**